src/osd/PeeringState.cc

```diff
@@ -4164,6 +4165,8 @@ void PeeringState::append_log(
 
   psdout(10) << __func__ << " approx pg log length =  "
 	     << pg_log.get_log().approx_size() << dendl;
+  psdout(10) << __func__ << " dups pg log length =  "
+	     << pg_log.get_log().dups.size() << dendl;
```

Comment:
How about dissecting this log entry into a tiny, dedicated commit? We could easily and safely backport it everywhere.

Reply:
+1!